packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -140,6 +146,16 @@ export class HbarLimitService implements IHbarLimitService {
     logger.info(
       `HBAR Limiter successfully configured: totalBudget=${totalBudget}, maxLimitForBasicTier=${HbarLimitService.TIER_LIMITS.BASIC}, maxLimitForExtendedTier=${HbarLimitService.TIER_LIMITS.EXTENDED}, maxLimitForprivilegedTier=${HbarLimitService.TIER_LIMITS.PRIVILEGED}, limitDuration=${limitDuration}, resetTimeStamp=${this.reset}.`,
     );
+
+    const activeUsersPerLimitPeriodCounterName = 'rpc_relay_hbar_rate_limiter_active_users';
+    this.register.removeSingleMetric(activeUsersPerLimitPeriodCounterName);
+    this.activeUsersPerLimitPeriodCounter = new Counter({
+      name: activeUsersPerLimitPeriodCounterName,
+      help: 'Relay Hbar rate limiter active users counter',
```

Comment:
note this isn't specific to the spending plan.
This should be in general, it just happens that eth_sendRawTransaction and others might spend hBAR.
Does have it in the spending plan mean we'd miss calls that don't spend HBAR.
Maybe this should be placed higher up in the server?